### DISCUSSION TITLE
Détection d'un virus avec l'importation auto-py-to-exe

### DISCUSSION THREAD

**Discussion by ProgrameurGentil** [brentvollebregt/auto-py-to-exe#418] [Q&A] [answered]
Bonsoir, 
Voulant transférer un programme python en format exécutable, j'ai utilisé l'importation auto-py-to-exe, après l'avoir vu sur plusieurs tutoriaux. Mais quand la convection fut finie, Avast m'a empêcher le virus   Win64:Trojan-gen  de s'introduire dans mon ordinateur.
J'ai également essayer l'importation pyinstaller mais cela a donné le même résultat. 

Je voudrais donc savoir si mon problème est normal et qu'est que je devrais faire pour le résoudre.

Merci d'avance pour vos réponse.
Cordialement,
                     ProgrammeurGentil.

**Comment by brentvollebregt** [comment, marked as answer]
In the help post, I discuss this under the heading ["My Antivirus Detected the exe as a Virus"](https://nitratine.net/blog/post/issues-when-using-auto-py-to-exe/#my-antivirus-detected-the-exe-as-a-virus).

Unfortunately there is nothing I can do about this but as long as the application you are bundling isn't a virus, you can ignore the warning.

**Reply by mostypc123** [reply]
but you can ask microsoft to be a verified app producer 👀

**Reply by brentvollebregt** [reply]
@mostypc123 I don't think that will stop it from being picked up by antivirus like Avast - it will stop Windows complaining though

**Reply by mostypc123** [reply]
i have eset and it doesnt complain about pyinstaller, only windows defender does
